src/Store/StoreThread.re

```diff
@@ -345,6 +345,25 @@ let start =
          })
       |> Option.value(~default=Isolinear.Sub.none);
 
+    let automaticallyChecksForUpdates =
+      Feature_AutoUpdate.Configuration.automaticallyChecksForUpdates.get(
+        config,
+      );
+    let licenseKey = Feature_AutoUpdate.Configuration.licenseKey.get(config);
+    let releaseChannel =
+      Feature_AutoUpdate.Configuration.releaseChannel.get(config);
+
+    let autoUpdateSub =
+      Service_AutoUpdate.Sub.autoUpdate(
+        ~uniqueId="autoUpdate",
+        ~licenseKey,
+        ~releaseChannel,
+        ~automaticallyChecksForUpdates,
+      )
+      |> Isolinear.Sub.map(msg =>
+           Model.Actions.AutoUpdate(Feature_AutoUpdate.Subscription(msg))
+         );
+
```

Comment:
We could probably consolidate this by having a 

`Feature_AutoUpdate.sub(~config)` - it could handle internally resolving the configuration settings, and then call `Service_AutoUpdate.sub` under the hood. This follows the pattern we have for several features, where the `Feature_XXX.sub` calls into maybe a bunch of `Service_XXX.sub`s

Reply:
Ah that works! I can definitely consolidate that. I based it off the `Service_Font` subscription, which I guess doesn't have a corresponding feature project.

Reply:
Ah that works! I can definitely consolidate that. I based it off the `Service_Font` subscription, which I guess doesn't have a corresponding feature project.

Reply:
Added in bb7b802!